views/stocksettings.blade.php

```diff
@@ -207,8 +207,21 @@ class="form-check-input custom-control-input user-setting-control"
 
 		@endif
 
+		<div class="form-group">
```

Comment:
Since this all affects only the /stockoverview page, the more appropriate group would be "Stock overview" and not "Common". Also a little help icon explaining shortly what this is about wouldn't hurt.